docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -1,11 +1,11 @@
 ---
```

Comment:
If you are going to do this, you need to also modify `docs/constants/navigation.js` to navigate to `expo-updates-1` instead of `expo-updates-0`

Reply:
We also will need to add redirect in:
* https://github.com/expo/expo/blob/main/docs/deploy.sh
* https://github.com/expo/expo/blob/main/docs/common/error-utilities.ts